src/inputs/RadioGroupField.tsx

```diff
@@ -38,6 +38,8 @@ export interface RadioGroupFieldProps<K extends string> extends Pick<Presentatio
   helperText?: string | ReactNode;
   onBlur?: () => void;
   onFocus?: () => void;
+  /** The group name for the radio group. */
+  name?: string;
```

Comment:
This fix makes a lot of sense, my only hesitation is that this page/component really isn't supposed to be used this way -- i.e. with random "drop this radio better whereever I want" custom layouts 😅 

Can we name this something like `unsupportedNameHack` and mention that this is only to be used for legacy pages, until we can move them over to a standard layout?